taichi/transforms/compile_to_offloads.cpp

```diff
@@ -90,6 +90,8 @@ void compile_to_offloads(IRNode *ir,
 
   // Final field registration correctness & type checking
   irpass::typecheck(ir);
+  irpass::fix_block_parents(ir);  // hot fix
```

Comment:
And this is actually only for fixing `OffloadedStmt`'s parents.

Reply:
All sounds good. Let's merge this in and deal with the rest later. Thanks again.